src/librustc_mir/build/matches/test.rs

```diff
@@ -35,9 +35,8 @@ impl<'a, 'gcx, 'tcx> Builder<'a, 'gcx, 'tcx> {
                 }
             }
 
-            PatternKind::Constant { .. }
-            if is_switch_ty(match_pair.pattern.ty) => {
-                // for integers, we use a SwitchInt match, which allows
+            PatternKind::Constant { .. } if is_switch_ty(match_pair.pattern.ty) => {
+                // for integers, we use a `SwitchInt` match, which allows
```

Comment:
Might as well capitalise and add the full stop to the sentence, while you're at it. :-)